src/frontend/src/lib/canisters/signer.errors.ts

```diff
@@ -33,3 +34,18 @@ export const mapSignerCanisterBtcError = (err: SignerCanisterBtcError): Canister
 	}
 	return new CanisterInternalError('Unknown SignerCanisterBtcError');
 };
+
+export const mapSignerCanisterSendBtcError = (
+	err: SignerCanisterSendBtcError
+): CanisterInternalError => {
+	if ('InternalError' in err) {
+		return new CanisterInternalError(err.InternalError.msg);
+	}
+	if ('PaymentError' in err) {
+		return new SignerCanisterPaymentError(err.PaymentError);
+	}
+	if ('BuildP2wpkhError' in err) {
+		return new CanisterInternalError(JSON.stringify(err.BuildP2wpkhError));
```

Comment:
We should probably create another mapper just for this particular error to improve the logging here, but for now it's fine to have it like this.